packages/components/modal/src/modal-content.tsx

```diff
@@ -21,6 +21,8 @@ export interface ModalContentProps extends AriaDialogProps, HTMLNextUIProps<"div
   children: ReactNode | ((onClose: () => void) => ReactNode);
 }
 
+const domAnimation = () => import("@nextui-org/dom-animation").then((res) => res.default);
```

Comment:
Wouldn't using LazyMotion enable lazy loading? I don't understand the need for this.
https://www.framer.com/motion/lazy-motion/

Reply:
as i rmb, `domAnimation` was loaded even it is not in use. e.g. before opening the modal. so i switched to load dynamically.

see https://www.framer.com/motion/guide-reduce-bundle-size/#lazy-loading